activity/activity/src/main/java/androidx/activity/EdgeToEdge.kt

```diff
@@ -93,9 +93,10 @@ fun ComponentActivity.enableEdgeToEdge(
                 EdgeToEdgeApi28()
             } else if (Build.VERSION.SDK_INT >= 26) {
                 EdgeToEdgeApi26()
-            } else if (Build.VERSION.SDK_INT >= 23) {
+            } else {
                 EdgeToEdgeApi23()
-            } else EdgeToEdgeApi21().also { Impl = it }
+            }
+    Impl = impl
```

Comment:
I've extracted the `Impl` assignment, otherwise ktlint reformat the `if` chain in a way that `also` only applied to the last element